bokehjs/src/lib/models/widgets/autocomplete_input.ts

```diff
@@ -149,8 +149,10 @@ export class AutocompleteInputView extends TextInputView {
           if (acnorm(text).startsWith(acnorm(value))) {
             completions.push(text)
           }
+          if (acnorm(text).includes(acnorm(value))) {
+            completions.push(text)
+          }
```

Comment:
Just adding this makes the previous clause pointless. Either the previous one should be removed, or the behavior should be made configurable, by adding a new property to `AutocompleteInput` model. The later seems a better idea, as we may introduce other matching algorithms in future (e.g. some sort of fuzzy matching).

Reply:
A configurable property is definitely needed, see https://github.com/bokeh/bokeh/pull/11889#issuecomment-992797211